assets/src/dashboard/constants/stories.js

```diff
@@ -123,15 +126,18 @@ export const STORY_STATUSES = [
   },
   {
     label: __('Published', 'web-stories'),
-    value: STORY_STATUS.PUBLISHED,
-    status: STORY_STATUS.PUBLISHED,
+    value: `${STORY_STATUS.PUBLISH},${STORY_STATUS.FUTURE}`,
```

Comment:
If I am not missing something, this should just work?

```suggestion
    value: PUBLISHED_AND_FUTURE,
```

Reply:
yup! you're right!